### PR TITLE
add march tech day email

### PR DESCRIPTION
**Pull request by Adimote**

### CODE REVIEW

File: SB2018/2018-03-15.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+## Food
+Food on campus is scarce, so we recommend either ordering in food, or bringing something with you. There are a few corner-shops located nearby however. This time the tech day is happening right next door to pret a manger and costa so it should be easier to grab a coffee or an expensive lunch.
+
+## Electronics
+As expected, you'll need to bring your robot to work on, in whatever state it may be. We also recommend bringing at least 1 laptop with you to work on. Laptops can be provided, however this needs to be arranged with us in advance.
+
+## Supervision
+Team members must be accompanied by any adult who is able to take responsibility for all of them. (this would ideally be a teacher, but it can be a parent if all the other parents agree).
+
+If you're planning on attending, be sure to let us know, so we know to expect you. If you're unable to make this tech day, there's not long to wait. Our next scheduled tech day is the 17th Feb!
```
Comment (RealOrangeOne) [inline SB2018/2018-03-15.md]: Date for next tech day is wrong here. Also, perhaps remove the 'be sure to let us know' stuff, it's a bit late for that ;)

File: SB2018/2018-03-15.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+> Southampton\
+> Hampshire\
+> SO17 1BJ
+
+There's a map attached showing where you should go to be let in. Its also on [google maps][maps-link]
+
+The day will run from 10:00am until 5:00pm, however feel free to come and go as you please.
+
+## Food
+Food on campus is scarce, so we recommend either ordering in food, or bringing something with you. There are a few corner-shops located nearby however. This time the tech day is happening right next door to both on-site Cafés, so it should be much easier to grab a coffee or an expensive lunch.
```
Comment (trickeydan) [inline SB2018/2018-03-15.md]: Where are these on-site Cafés? Are there secret ones within Zepler?

File: SB2018/2018-03-15.md (hunk 1 above)
Comment (Adimote) [reply SB2018/2018-03-15.md]: fixed in 6889b077edd979164bbf57f7277439628aa2ac8b

File: SB2018/2018-03-15.md (hunk 2 above)
Comment (Adimote) [reply SB2018/2018-03-15.md]: fixed in d9cd5c5b3c3835785570ad269dc89db56921612f

File: SB2018/2018-03-15.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+---
+to: All Teams
+subject: Tech day Location Change - This Saturday 17th March
```
Comment (RealOrangeOne) [inline SB2018/2018-03-15.md]: Don't specify location change, as we never said it was in 25. Keep same subject lines as last emails

**Review by RealOrangeOne — APPROVED** [review]

File: SB2018/2018-03-15.md (hunk 2 above)
Comment (trickeydan) [reply SB2018/2018-03-15.md]: I've just realised that you're referring to Costa and Pret...

**Pull request merged**